lib/checkout.rb

```diff
@@ -0,0 +1,28 @@
+require_relative 'menu'
```

Comment:
This class is implemented nicely, methods are short and it is clear what each one does and is responsible for